src/AlertBox/AlertBox.jsx

```diff
@@ -0,0 +1,70 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PureComponent, PropTypes} from "react";
+
+import "./AlertBox.less";
+import {CloseIcon, WarningIcon, SuccessIcon, ErrorIcon, InfoIcon} from "./Icons";
```

Comment:
since `Icons` isn't actually a React component, but just a file that exports other react components, I'd lowercase it and call the file `icons.jsx`.